web/satellite/tests/unit/referralStats/referralStats.spec.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import { createLocalVue, mount } from '@vue/test-utils';
+import Vuex from 'vuex';
+import ReferralStats from '@/components/referral/ReferralStats.vue';
+import { CreditsApi, CreditUsage } from '@/types/credits';
+import { UpdatedUser, User, UsersApi } from '@/types/users';
+import { makeCreditsModule } from '@/store/modules/credits';
+import { makeUsersModule } from '@/store/modules/users';
+import { USER_ACTIONS } from '@/utils/constants/actionNames';
+
+const {
+    GET,
+} = USER_ACTIONS;
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+const mockUser = new User('1', 'full name', 'short name');
+const mockCredits = new CreditUsage(1, 2, 3);
+
+class UsersApiMock implements UsersApi {
+    get(): Promise<User> {
+        return Promise.resolve(mockUser);
+    }
+
+    update(user: UpdatedUser): Promise<void> {
```

Comment:
I don't think we are going to update it. The update will be triggered on the backend

Reply:
its mock for users api. we need this dependency to initialize usersModule(store), because referralStats component use both users and credits modules